### PR TITLE
chore: remove engine-v1

### PR DESCRIPTION
**Pull request by pimeys**
💥 

- Removes all forks (except the multipart-stream which hasn't been updated in three years)
- Removes engine-v1 and everything related to it

Left to do:

- Rename engine-v2 to engine
- Consider the serialize impls of ours in config and schema?

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -37,32 +27,8 @@ exclude = [
 ]
 
 [patch.crates-io]
-axum = { git = "https://github.com/grafbase/axum", rev = "c3146f9ca921907d9884fbf7549a1520e9e72eac" }                                 # axum-tungstenite-upgrade
 multipart-stream = { git = "https://github.com/grafbase/multipart-stream-rs-fork", rev = "06ff198e4041c8a8c1c93e580c260d597727c193" } # http-1.0-fix-multipart-mixed
```
Comment (pimeys) [inline Cargo.toml:30]: We have to upstream these changes. One is for HTTP 1.0 crate and another one a fix from graeme.

File: engine/crates/integration-tests/src/openid.rs [hunk 2]
```diff
@@ -93,13 +93,10 @@ impl<
         GC,
         JE,
         JS,
-        JT,
-        JU,
```
Comment (tomhoule) [inline engine/crates/integration-tests/src/openid.rs:97]: heh

File: engine/crates/integration-tests/tests/federation/basic/headers.rs [hunk 3]
```diff
@@ -4,160 +4,6 @@ use engine_v2::Engine;
 use graphql_mocks::EchoSchema;
 use integration_tests::{federation::EngineV2Ext, runtime};
 
-#[test]
-fn test_default_headers() {
```
Comment (tomhoule) [inline engine/crates/integration-tests/tests/federation/basic/headers.rs:8]: Are there toml equivalents for these? (I can't read everything super carefully so may have missed them)

**Review by tomhoule — COMMENTED** [review]
a few more hundred files, getting closer

File: engine/crates/integration-tests/tests/integration_tests.rs [hunk 4]
```diff
@@ -1,16 +1,3 @@
...
-mod execution;
 mod federation;
-mod graphql_connector;
-mod mongodb;
-mod openapi;
-mod partial_caching;
-mod postgres;
-mod rate_limiting;
-mod subgraph;
-mod tracing;
```
Comment (tomhoule) [inline engine/crates/integration-tests/tests/integration_tests.rs:16]: :fire:

**Review by tomhoule — APPROVED** [review]
nice! good job on porting the integration tests

**Pull request merged**